### DISCUSSION TITLE
Relative path in build

### DISCUSSION THREAD

**Discussion by sgdot** [saeghe/saeghe#35] [Ideas]
Hi!

I think that relative paths in the build are better and more portable than absolute paths

And also be possibble build saeghe with saeghe

**Comment by MortezaPoussaneh** [comment]
Could you please explain more about how it can improve portability?

**Reply by sgdot** [reply]
Now you cannot copy and run the builded package from another directory

**Reply by MortezaPoussaneh** [reply]
I didn't use relative paths because I wanted the import parts to be as fast as possible for both, build and runtime. For the build, it adds extra execution time for making a relative path for functions. For both, classes and functions, having a relative path adds some execution time in the runtime.
But in general, I liked your idea, it seems to have many benefits.
I will think about it after 2 new features that I'm working on them currently :wink: 
Thank you very much. :pray:
